app/src/main/java/org/jellyfin/androidtv/util/KeyUtils.kt

```diff
@@ -0,0 +1,24 @@
+package org.jellyfin.androidtv.util
+
+import android.view.KeyEvent
+
+object KeyUtils {
+	@JvmStatic
+	fun isSelect(code: Int) = code == KeyEvent.KEYCODE_DPAD_CENTER || code == KeyEvent.KEYCODE_ENTER
+
+	@JvmStatic
+	fun isNumber(code: Int) = code >= KeyEvent.KEYCODE_0 && code <= KeyEvent.KEYCODE_9
+
+	@JvmStatic
+	fun isChannelSurfUp(code: Int) = code == KeyEvent.KEYCODE_PAGE_UP || code == KeyEvent.KEYCODE_CHANNEL_UP
+	@JvmStatic
```

Comment:
Spacing is inconsistent in this file. I would prefer a blank line before each annotated function like was done above.